# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoConduit.h
# Commit: a1c16b86d3a7
# Full Hash: a1c16b86d3a7f328a690c00f86b837a8a67a656c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Remove RtcpEventObserver. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102287
# ==============================================================================

diff -r c7dbdd82e8e8 -r a1c16b86d3a7 dom/media/webrtc/libwebrtcglue/VideoConduit.h
--- a/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Mon Feb 01 09:28:23 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Mon Jan 18 16:41:26 2021 +0100
@@ -269,8 +269,6 @@
 
   void CollectTelemetryData() override;
 
-  void SetRtcpEventObserver(mozilla::RtcpEventObserver* observer) override;
-
  private:
   // Don't allow copying/assigning.
   WebrtcVideoConduit(const WebrtcVideoConduit&) = delete;
@@ -477,9 +475,6 @@
   // Accessed only on mStsThread
   Maybe<DOMHighResTimeStamp> mLastRtcpReceived;
 
-  // Accessed only on main thread.
-  mozilla::RtcpEventObserver* mRtcpEventObserver = nullptr;
-
   // Tracking the attributes of received frames over time
   // Protected by mTransportMonitor
   dom::RTCVideoFrameHistoryInternal mReceivedFrameHistory;